Scripts/Runtime/Core/Steps/CallMethodAnimationStep.cs

```diff
@@ -10,8 +10,11 @@ public class CallMethodAnimationStep : AnimationStepBase
         private UnityEvent methodToCall;
         
         public override string DisplayName => "Method Call Step";
-        
-        public override float Duration { get; }
+
+        public override float Duration
+        {
+            return 0;
```

Comment:
It's missing the get, right? 
It should be 
`public override string Duration => 0;`

Reply:
Yes! Forget to push this in the previous commit. 
Should be OK now. :D